.github/workflows/test-application.yaml

```
@@ -0,0 +1,98 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - develop
+            - '[0-9]+.[0-9]+'
```

i remove the `master` and `develop` branch, i am sorry 😇 

```suggestion
            - '[0-9]+.x'
            - '[0-9]+.[0-9]+'
```